specs/validator/tableland-openapi-spec.yaml

```diff
@@ -122,52 +122,29 @@ paths:
                 $ref: "#/components/schemas/TxnReceipt"
         "400":
           description: Invalid query/statement value
-  /tables/structure/{hash}:
+  /tables/{chainId}/{tableId}:
     get:
       tags:
         - tables
-      summary: Finds tables by structure
-      description: Returns all tables that match a given structure
-      operationId: findTablesByStructure
+      summary: Get information about a table
+      description: Returns information about a single table, including schema information
+      operationId: getTableById
       parameters:
-        - name: hash
+        - name: chainId
           in: path
-          description: Structure hash to filter on
+          description: The parent chain to target
```

Comment:
I used the same `description: ` as other `{{chainId}}` APIs.
I'm not sure if "parent" is a word we need to use here?

In any case, after we have everything more stabilized, we can do a complete pass again to improve descriptions, etc. Just mentioning that this sounded weird to me.